index.d.ts

```diff
@@ -295,6 +300,17 @@ declare namespace execa {
 		Promise<ExecaReturnValue<StdoutErrorType>>;
 }
 
+interface killOptions {
+	/**
+	If the first signal does not terminate the child after a specified timout, a `SIGKILL` will be sent to the process.
+	*/
+	retry: boolean;
+	/**
+	How long to wait for the child process to terminate before sending `SIGKILL`.
```

Comment:
```suggestion
	Milliseconds to wait for the child process to terminate before sending a `SIGKILL` signal.
```

Reply:
You also need to document the default values for these options, per https://github.com/sindresorhus/typescript-definition-style-guide

Reply:
Done.

Reply:
I think you forgot to add the default value in the TypeScript definition.